bin/run_tests

```diff
@@ -51,11 +54,15 @@ while getopts "dpsh" opt; do
 	s)
             DIRECT_MODEL_SGX_TESTS=1
             ;;
+	x)
+	    FABRIC_PROXY_MODEL_SGX_TESTS=1
+	    ;;
         h)
             echo "Usage: run_tests script to trigger tests for direct/proxy model"
             echo "    run_tests -d       To run direct model tests."
             echo "    run_tests -p       To run proxy model tests."
             echo "    run_tests -s       To run direct model tests in SGX H/W mode."
+	    echo "    run_tests -x       To run (Singleton) fabric proxy model SGX tests."
```

Comment:
The indentation is wrong. Change the tab to spaces.

Reply:
Done